Pod/Classes/AwesomeMediaControl.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  AwesomeMediaControl.swift
+//  Pods
+//
+//  Created by Gabriel Coman on 10/12/2018.
+//
+
+import Foundation
+import AVFoundation
+import AVKit
+
+@objc(SAAwesomeMediaControl) public class AwesomeMediaControl: AVPlayer, MediaControl {
+    
+    private let VIDEO_END_NAME = NSNotification.Name.AVPlayerItemDidPlayToEndTime
+    private let VIDEO_END_SEL = #selector(didFinishPlaying(_:))
+    private let PERIODIC_TIME_INTERVAL = CMTime(seconds: 1, preferredTimescale: 2)
```

Comment:
I don't like seeing constants written this way in swift, totally different to the way T and I do things.
```suggestion
    private let periodicTimeInterval = CMTime(seconds: 1, preferredTimescale: 2)
```